zippkg/helpers.go

```diff
@@ -118,15 +117,15 @@ func Decode(f *zip.File, dest interface{}) error {
 	return nil
 }
 
-// AddFileFromDisk reads a file from disk and adds it at a given path to a zip file.
-func AddFileFromDisk(z *zip.Writer, zipPath, diskPath string) error {
+// AddFileFromStorage reads a file from internal storage and adds it at a given path to a zip file.
+func AddFileFromStorage(z *zip.Writer, zipPath, storagePath string) error {
```

Comment:
Revert name change and add note:
// TODO: Rename to AddFileFromStorage in next major version release (v2).

Reply:
and // NOTE: If disk storage cannot be used, memory storage can be used instead by calling memstore.SetAsStorage()

Reply:
fixed